queue_services/account-mailer/src/account_mailer/email_templates/common/style.html

```diff
@@ -4,8 +4,7 @@
mp3yJDZow8xf+EE3muUZYm9A5zTnKH2pqbR+6NmrcahdBCQYXuXv4jcPOen3vRviPXzSuz5ovsbZ7wsjlGvNLUPrO574A/2/hkvrA/boA9F60ULHhmjV/ktH3bDueomGi+2Y72gZ+4gdTO674G3imnrRX0n9GCT+jcY35IVq/+WD7sx71m/v7gDemE9Y/RnUfOf4LqZSV1ESVFb9Yn3b3AdR3kIOMtkvA/WGR8wXmXPfAcxGigNECdEyiBAQb4zEjWJAj146k3SE56VKs9UUTT/yzlzXg7D+izzz3znCGdKI/0fubjYcQAAeNpjYGDQgcIShmWMXUxyTNuYvZgzmCcx72J+wqLG4seSwzKDZRMrE6sBawfrFzYPti3sauxe7Jc4XDiaOI5w3OD4xcnH2cTlwlXD9YLbiLuAew33I54Ang6eQzyfeDV4/XgX8L7gs+Nr4nvB78C/iv+WgISAk8ApgT+CWoIRgm2CCwSvCH4S4hIyEdokLCQcIbxPREmkTOSHqJNolegRMRYxBbE+sRviauJ94t8kjCTmSPyT9JEsk7wnJSDlI1UldUXqg7SPdBcQnpCxkpknKyJbJjtLzkSuRJ5PvkF+ifwZ+VcKsxQ2KfxT9FGcpPhDSUmpTumY0idlFWUP5T7lLSpmKltUm1R/qTmotaldUPdQ36OholGhyaAZoDlL84NWitYWbRbtCO1jOlo663RFdKt0n+hV6XPp6+iv0P9m4GYww1DAMMBwjpGR0QHjIONnJutMs8yUzC6ZB5gfsbCy2GEZZdllucvynZWN1SZrE+tJ1h9somxe2WbZrrGTs6uxu2fvZX/JQc6hweGTY5YTh1OW0xSnRzjgB6d/zjzOUs4aznbOIc5Zzh3Oy5yPON9y4QFCJRcLIKxwWeWyzeWdq5RrgesOACiCmLEAAAABAAAA6gBJAAUAAAAAAAIAAQACABYAAAEAAVEAAAAAeNrFVc1OU0EU/tpi+VG7MsQYY25cGmkpAkoxJvwIVpEaihKXl/ZSiG1v7b1V2LnwCVz4AK6NceFDKL6AvoAL49qFK79z7qF/QYQVmdyZb+b8zDfnnJkL4AJ+I4HYwDCAfX4RjiHFWYTjxN8MJ/AePwwP4ErsjeEz2I59NJzE5fhFw4NYjE8aHsK1+K7hYdyPvzM8gtfxX4bP4k6iYfgcbic+GD4ff5v4aTiFyeQrw58xmvxk+AvGkwec95FK/jH8FanBoQh/T+DS4CgW4KOBPTSxgwq2EcLBBMaRxRTRMqU+16vwOMujjhLSRHNcqXJca1sFOvM4evT1gn2ZmiOH+n/EmU+dHfUnnn1ssZ+nxYL6jxjMnGivedo6KMKl10DlFbRo6VLniWoG9OBTKv7T3EHaLM9YYFsmqlMa8hujpk/bprLLGZub7Gc0LjmO45RscZzBDbZblJb6vAmfAzZjfXxmzec00RJWyC9PnWmuTPadwzmlkxzFftW8O2QhDF1+Iddd5sFDTXWetbN6eA09VP7iZ4/1EUnEsqJVUTlSc1FzL9UgEVjnzKXtyH/rqttOasbBJn06h+xQ7tkhtB2knkW3TMkWv24fISvTO0ZtS/2G3CWHDNtLbWnu04lRmnY+d8tw7lE3085m5h/WtT7+HQ8B11rK12ecti1fS6ofapUJu5BeJGdeO6NVjsK+rndMTtkiLmvtdc5aZI5WOBZ013qP55UeD9e50l+5WfLM6g0/CbOyjqG+J5tkFRq/yKer/VW+GUWtoCKxxH5OuRaVxwbROu6R9WOOMp9jfa+xX+U8j7tqW+CKw4wVuLqoFnnFkWxJ36VVPOX4gBLREd8eWUXRaepsl5Fpai0FyrGp56hxtWr1k9azenrCk8fVYYz8npwEalOyCi1pLl29US29XVEuGsqwprE8yEhg8Stb/mt6FpdfR17R2hbbevue7tkLIzUSdFV8eIys9t+HgIwlsw195dPKrcpRzlihvKCv5QYtN/XkkdesvoFSRS2LZkNXp6zPUTrNXl6+6N8ywVbWd8/VVy20iAh/OUU34+dc2aFM+Fb/AtbqTP0AAAB42m3QR0yTcRjH8e8DpYWy98a9V9+3LcPdAq97b3GhQFtFwGJVXGjcMxoTPWlcFzXuGY16UONGjSPqwbM7HtSrFt6/N5/LJ8+TPE+e/Iigrf64qOF/9RkkQiKJxEIUVmxEE4OdWOKIJ4FEkkgmhVTSSCeDTLLIJodc8singHa0pwMd6URnutCVbnSnBz3pRW/60Jd+ONDQceLCTSFFFFNCfwYwkEEMZghD8eCllDLKMRjGcEYwklGMZgxjGcd4JjCRSUxmClOZxnRmMJMKZjGbOcxlHpVi4Rgb2cRN9vORzexmBwc5wXGJYjvv2cA+sYqNXRxgK3f4INEc4iS/+MlvjnKah9znDPNZwB6qeEw1D3jEM57wlBY+hdN7yXNecBYfP9jLG17xGj9f+MY2FhJgEYuppY7D1LOEBoI0EmIpy1geTnkFK2liFWtYzTWO0Mxa1rGer3znOuc4zw3e8k5ixC6xEifxkiCJkiTJkiKpkibpksEFLnKFq9zlEpe5xxZOSSa3uC1Zks1OyZFcyZN8KbD6apsa/JotVBdwOBxlph6HUvVeXelUlrSqhxeUmlJXOpUupVtZqCxSFiv/3fOYauquptlrAr5QsLqqstFvjnTD1G1YykPB+rbGbZS2anjNP8LqSqfS9Rd66Zz9AAAAeNpFzU0OwVAUBeC+Vv9Q+ouJpExI3grEVDsxEYmkTSQ2YMyQSYes5dZIxIpsglOe19n9Ts7NubN3QeyiLMlaZSVj17xMDZ6NyM2XFK5xnPMhGXybKaTFCWl8Qa04uWkvlX/RBFqFgA00dwIWYKcCJmDNBAzAjAV0wAgEGoC+EGhXO/4PjByxHiB1niovtfQA+mCwl/RAfyrpxslD8dhckUkXBXco2QG7nmRY9Z3Tu+5HVRKycZ308BIdJftgbyM5APuTP3MK+QcE5WOG) format('woff');
```

Comment:
Didn't we try this one?

Reply:
yeah, but I was thinking that it was possibly the extra font files in account-mailer that were actually making the emails larger so I wanted to test if just removing those would fix the problem.

Reply:
The problem is it's not a font published any cached CMS, so folks will get aliased fonts when viewing online.
I'm not sure why we use it tbh, for printing we had to change the font as the gov't print services have issues with it.

Reply:
The other option is just to install the font on the print server and ref it, and let the font aliasing issue be another issue to take care of.